.github/workflows/kata-deploy-push.yaml

```diff
@@ -0,0 +1,56 @@
+name: kata-deploy-build
+
+on: push
+
+jobs:
+  build-asset:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        asset:
+          - kernel
+          - shim-v2
+          - qemu
+          - cloud-hypervisor
+          - firecracker
+          - rootfs-image
+          - rootfs-initrd
+    steps:
+      - uses: actions/checkout@v2
+      - uses: docker-practice/actions-setup-docker@master
```

Comment:
Non official action, is it safe?

Reply:
fwics this is: https://github.com/docker-practice/actions-setup-docker.

If we can't use a GitHub official action, I think the best we can do is review the code and if >1 person is happy with that, pin the version: `@master` isn't ideal as it could change constantly.

Reply:
+1 for pinning the version.

Reply:
Other option is to fork it. This is what I did with prior actions.

Reply:
I have replaced the action, instead I use the docker installer script.